appcues/src/main/java/com/appcues/debugger/ui/DebuggerFloatingActionButton.kt

```diff
@@ -52,70 +52,77 @@ internal fun BoxScope.DebuggerFloatingActionButton(
     debuggerState: MutableDebuggerState,
     debuggerViewModel: DebuggerViewModel,
 ) {
-    val resizeBy = animateFloatAsState(
-        when {
-            // if its dragging and paused we diminish so it doesn't mess with current RESUMED debugger
-            // important for our case where we have transparent activity on top of customer's
-            // and we know we have two debuggers showing at the same time. one on top of the other.
-            debuggerState.isDragging.targetState && debuggerState.isPaused.value -> FAB_PAUSED_SIZE_MULTIPLIER
-            debuggerState.isDragging.targetState -> FAB_DRAGGING_SIZE_MULTIPLIER
-            else -> FAB_DEFAULT_SIZE_MULTIPLIER
-        }
-    )
-    AnimatedVisibility(
-        visibleState = debuggerState.isVisible,
-        enter = scaleIn() + fadeIn(),
-        exit = scaleOut() + fadeOut(),
-        modifier = Modifier
-            .align(Alignment.TopStart)
-            // this is used along with the dynamic size of FAB Box below
-            // so we create the nice effect of enlarging equally to all directions
-            .offsetResize(
-                offset = debuggerState.getFabPositionAsIntOffset(),
-                originalSize = debuggerState.fabSize,
-                resizeBy = resizeBy.value,
-            )
-    ) {
-        val elevation = animateDpAsState(
-            targetValue = when {
-                debuggerState.isDragging.targetState -> 8.dp
-                debuggerState.isExpanded.targetState -> 2.dp
-                debuggerState.isVisible.currentState -> 4.dp
-                else -> 0.dp
-            }
-        )
+    val resizeBy = resizeAnimatedState(debuggerState)
 
-        Box(
+    // only draw if IntOffset is not null
+    debuggerState.getFabPositionAsIntOffset()?.let { offset ->
```

Comment:
also I'm hiding the whole FAB if IntOffset is not valid, this is to avoid some glitch I found while opening Flows